VideoApp/VideoApp/ViewControllers/Settings/SDKLogLevel/SelectCoreSDKLogLevelViewModel.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Copyright (C) 2020 Twilio, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+class SelectCoreSDKLogLevelViewModel: BaseSelectSDKLogLevelViewModel, SelectOptionViewModel {
+    let title = "Core"
+    var selectedIndex: Int {
+        get { SDKLogLevel.allCases.firstIndex(of: appSettingsStore.coreSDKLogLevel) ?? 0 }
+        set { appSettingsStore.coreSDKLogLevel = SDKLogLevel.allCases[newValue] }
```

Comment:
I don't love that this logic is duplicated several times. I experimented with making it more DRY. I created a reusable view model but it ended up moving some logic to the factory that builds the view model and factories shouldn't really be doing this work. So didn't really come up with anything great. I did create `BaseSelectSDKLogLevelViewModel` to reuse a little. 

This is kind of an unusual situation where we have multiple very similar settings so probably fine.